SPECS/python-rpm-spec/python-rpm-spec.spec

```diff
@@ -0,0 +1,56 @@
+# sitelib for noarch packages, sitearch for others (remove the unneeded one)
+%{!?__python2: %global __python2 %__python}
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%{!?python2_sitearch: %global python2_sitearch %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
+
+%if 0%{?fedora}
```

Comment:
Please remove all macros that are specific to other distros. For Fedora ones, assume that `%if 0%{?fedora}` evaluates to a true statement.

Reply:
Done